src/components/ForecastSummary.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import PropTypes from "prop-types";
+import WeatherIcon from "react-icons-weather";
+
+function ForecastSummary(props) {
+  const { date, temperature, description, icon, onSelect } = props;
```

Comment:
You could destructure props directly in the component definition and remove the destructuring from `props` on the next line:
```
function ForecastSummary({ date, temperature, description, icon, onSelect }) {
```